commitizen/cmd.py

```diff
@@ -12,6 +12,14 @@ class Command(NamedTuple):
     return_code: int
 
 
+def _try_decode(bytes_):
```

Comment:
could you add the type hint here? `bytes_: bytes`?

Reply:
Yup, I didn’t know that the code is required to have type hints 👍🏼

Reply:
@Woile I amended the initial commit.

What’s the general recommendation for this repo — 
- any number of PR commits and you’ll squash on merge, or 
- careful/useful commits and you’ll merge the PR with its history, or
- keep amending the initial commit?

Reply:
A combination of the last 2. Think which commits you want to be reflected in the changelog. As of now is good 👍🏻